central/metadata/service/service_impl.go

```diff
@@ -51,9 +51,10 @@ func (s *serviceImpl) AuthFuncOverride(ctx context.Context, fullMethodName strin
 // GetMetadata returns the metadata for Rox.
 func (s *serviceImpl) GetMetadata(ctx context.Context, _ *v1.Empty) (*v1.Metadata, error) {
 	metadata := &v1.Metadata{
-		BuildFlavor:   buildinfo.BuildFlavor,
-		ReleaseBuild:  buildinfo.ReleaseBuild,
-		LicenseStatus: v1.Metadata_VALID,
+		BuildFlavor:      buildinfo.BuildFlavor,
+		ReleaseBuild:     buildinfo.ReleaseBuild,
+		LicenseStatus:    v1.Metadata_VALID,
+		IsManagedService: env.ManagedCentral.BooleanSetting(),
```

Comment:
Be wary of the compatibility tests.  We ran into this a while back when I tried to update this endpoint.  See https://issues.redhat.com/browse/ROX-14188
Fix to sensor was delivered in 3.74, but this will very likely break any sensor prior to 3.74.